src/components/instantlaunches/launch/index.js

```diff
@@ -29,13 +30,19 @@ import ErrorTypography from "components/error/ErrorTypography";
 import DEErrorDialog from "components/error/DEErrorDialog";
 
 const InstantLaunchStandalone = (props) => {
-    const { id: instant_launch_id, showErrorAnnouncer } = props;
+    const {
+        id: instant_launch_id,
+        resource: resource_path,
+        showErrorAnnouncer,
+    } = props;
     const [config] = useConfig();
     const [userProfile] = useUserProfile();
     const [computeLimitExceeded, setComputeLimitExceeded] = useState(
         !!config?.subscriptions?.enforce
     );
     const [errorDialogOpen, setErrorDialogOpen] = useState(false);
+    const [resource, setResource] = useState(!!resource_path ? null : {});
```

Comment:
I'm unsure this is strictly necessary, could possibly just pass in `{}`. But I think it's more likely to error (as it should) if when a path is passed in it sets this to null.